hyrax/app/renderers/nested_person_attribute_renderer.rb

```diff
@@ -67,13 +67,13 @@ def attribute_value_to_html(input_value)
         label = 'Role'
         val = v['role'][0]
         term = RoleService.new.find_by_id(val)
-        val = term['label'] if term.any?
+        val = term['id'] if term.any? #using id as proxy for English-only text
```

Comment:
note: ``id`` comes from this line https://github.com/antleaf/nims-hyrax/blob/i422-restore-contact-person/hyrax/config/authorities/roles.yml#L17